packages/core/src/modules/oob/OutOfBandApi.ts

```diff
@@ -787,6 +788,13 @@ export class OutOfBandApi {
 
     for (const invitationDid of outOfBandInvitation.invitationDids) {
       const connections = await this.connectionsApi.findByInvitationDid(invitationDid)
+
+      // Also search for legacy invitationDids based on inline services (TODO: remove in 0.6.0)
+      for (const service of outOfBandInvitation.getInlineServices()) {
+        const records = await this.connectionsApi.findByInvitationDid(outOfBandServiceToInlineKeysNumAlgo2Did(service))
+        connections.push(...records)
+      }
```

Comment:
Not sure we should do this loop in the other loop? As it will run multiple times?